DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -70,8 +70,8 @@ public void ConfigureServices(IServiceCollection services)
                 options =>
                 {
                     options.AddPolicy(
-                        CustomPolicies.UserOnly,
-                        policy => CustomPolicies.ConfigurePolicyUserOnly(policy)
+                        CustomPolicies.BasicUser,
+                        policy => CustomPolicies.ConfigurePolicyBasicUser(policy)

```

Comment:
I have added CustomPolicies.UserOnly usage to the ChangePasswordController as part of 890, so we need to be careful that this change doesn't break that one when merging into uar-test

Reply:
I suspect this might be merged first as it is the smaller PR, but will just have to check when it comes to merging.